helm_deploy/values-dev.yaml

```diff
@@ -16,8 +16,8 @@ ingress:
 
 env:
   HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/
-  HMPPS_COOKIE_NAME: hmpps-session-dev
+  COMMUNITY_API_URL: https://community-api.test.delius.probation.hmpps.dsd.io
```

Comment:
did we get any clarity on this vs the `secure` version? i looked at my local configs and i generally use the latter.

Reply:
No, good point, I still need to do that.

Reply:
Other apps use the secure one, so I'll go with that.

Reply:
done in c3977c82679022a50fa56f9c75428f2a94fcd5d2